### PR TITLE
[RFR] Improved the way we deal with color preview and sidebar collapsing

### PR DESCRIPTION
**Pull request by HugoGiraudel**

### CODE REVIEW

File: scss/base/_layout.scss [hunk 1]
```diff
@@ -5,7 +5,7 @@
   display: none;
 
   @include respond-to('medium') {
-    @include absolute(top 10px left 10px);
+    @include absolute(top 8px left 20px);
```
Comment (kaelig) [inline scss/base/_layout.scss]: It's surprising to have a mixin for that. Any particular reasons?

Comment (HugoGiraudel) [inline scss/base/_layout.scss]: Shorter syntax.

Comment (kaelig) [inline scss/base/_layout.scss]: Fair enough!

Although…
```scss
@include absolute(top 8px left 20px);
position: absolute; top: 8px; left: 20px;
```

I think I'd pay for those 4 extra chars to keep code close to the metal.

Comment (HugoGiraudel) [inline scss/base/_layout.scss]: This would break pretty much any linter.

Comment (HugoGiraudel) [inline scss/base/_layout.scss]: That being said, the mixin is crap. I mean, it's clever but this is not how I deal with position anymore. I'll change it to something like this:

```scss
@mixin position($position, $top: null, $right: null, $bottom: null, $left: null) {
  position: $position;
  top: $top;
  right: $right;
  bottom: $bottom;
  left: $left;
}

@mixin absolute($args...) {
  @include position(absolute, $args...);
}
```

```scss
.foo { 
  @include absolute($top: 8px, $left: 20px);
}
```

Comment (HugoGiraudel) [inline scss/base/_layout.scss]: Just pushed it on `develop`: https://github.com/SassDoc/sassdoc-theme-default/commit/e25bca2250a0e637d23711d2f1db4bc6cf452430.

**Pull request merged**